ceph/ceph/templates/deployment-mgr.yaml

```diff
@@ -80,15 +80,18 @@ spec:
           ports:
             - containerPort: {{ .Values.network.port.mgr }}
           livenessProbe:
-              httpGet:
-                path: /
-                port: {{ .Values.network.port.mgr }}
-              initialDelaySeconds: 120
+              exec:
+                command:
+                  - sh
+                  - /check_mgr.sh
+              initialDelaySeconds: 30
               timeoutSeconds: 5
           readinessProbe:
-              httpGet:
-                path: /
-                port: {{ .Values.network.port.mgr }}
+              exec:
+                command:
+                  - sh
+                  - /check_mgr.sh
+              initialDelaySeconds: 30
               timeoutSeconds: 5
```

Comment:
This does this every periodSeconds, which is 10 by default; that seems kinda frequent

Reply:
I'm not sure it is. The probe has to fail 3 times for the Pod to recreate, that leaves a 39s max interval between failure and recreation (more if no quorum, but you have other issues in that case).

Reply:
fair enough.